packages/tldraw/src/lib/shapes/shared/freehand/svgInk.ts

```diff
@@ -115,13 +115,14 @@ function cleanUpPartition(partition: StrokePoint[]) {
 	if (partition.length > 1) {
 		partition[0] = {
 			...partition[0],
-			vector: Vec.FromAngle(Vec.Angle(partition[1].point, partition[0].point)),
```

Comment:
Generally working with angles is slower than working with vectors. I made a few changes like this in this PR, the rest are mostly drivebys.